src/audio/google_rtc_audio_processing.c

```diff
@@ -34,8 +34,18 @@
 
 #include <google_rtc_audio_processing.h>
 #include <google_rtc_audio_processing_platform.h>
+#include <google_rtc_audio_processing_sof_message_reader.h>
 
-#define GOOGLE_RTC_AUDIO_PROCESSING_SAMPLERATE 48000
+#define SAMPLE_RATE_HZ 48000
+#define SAMPLES_PER_CHANNEL (SAMPLE_RATE_HZ / 100)
```

Comment:
I would be careful with these generic names, best to use the prefix to prevent collisions with global names.

Reply:
Thanks for the headsup! I've now added the GOOGLE_RTC_AUDIO_PROCESSING prefix to avoid that